packages/cli/src/commands/prerender.js

```diff
@@ -47,15 +50,108 @@ const mapRouterPathToHtml = (routerPath) => {
   }
```

Comment:
I think this file needs to be merged with main. Most of the code here probably just needs to move to the prerenderHandler.js file

Reply:
Done